rele/subscription.py

```diff
@@ -2,7 +2,6 @@
 import logging
 import time
 
-import rele
```

Comment:
How come linting did not catch this before?

Reply:
😞 we dont have isort on this project. Ill submit another PR for this.